.github/workflows/tools.yml

```diff
@@ -31,81 +49,114 @@ jobs:
-     #---------- Ontology Explorer ----------#
-    - name: Ontology Validator Install dependencies
-      run: |
-        python setup.py install
-      working-directory: ./tools/validators/ontology_validator
-    - name: Instance Validator Install dependencies
-      run: |
-        python -m pip install .
-      working-directory: ./tools/validators/instance_validator
-    - name: Ontology Explorer Install dependencies
-      run: |
-        python setup.py install
-      working-directory: ./tools/explorer
-    - name: Run Pylint on Ontology Explorer
-      run: |
-        pylint --rcfile ./styles/pylintrc ./tools/explorer
-    - name: Ontology Explorer Tests
-      run: |
-       python -m unittest discover -p '*_test.py' --failfast -v
-      working-directory: ./tools/explorer
+
     #---------- Ontology Yaml Type Validator ----------#
     - name:  Ontology Yaml Type Validator Install dependencies
       run: |
         python setup.py install
       working-directory: ./tools/validators/ontology_validator
+    - name: Run pylint on Ontology Type Validator
+      if: steps.changes.outputs.type_validator == 'true'
+      run: |
+        pylint --rcfile ./styles/pylintrc ./tools/validators/ontology_validator
     - name: Ontology Yaml Validator Generator Tests
+      if: steps.changes.outputs.type_validator == 'true'
       run: |
        python -m unittest discover -p '*_test.py' --failfast -v
       working-directory: ./tools/validators/ontology_validator
+
     #---------- Ontology Yaml Instance Validator ----------#
     - name: Instance Yaml Validator Install dependencies
       run: |
         python -m pip install .
       working-directory: ./tools/validators/instance_validator
-    - name: Run Pylint on Instance Validator
+    - name: Run pylint on Ontology Instance Validator
+      if: steps.changes.outputs.instance_validator == 'true'
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/validators/instance_validator
     - name: Instance Yaml Validator Tests
+      if: steps.changes.outputs.instance_validator == 'true'
       run: |
        python -m unittest discover -p '*_test.py' --failfast -v
       working-directory: ./tools/validators/instance_validator
+
+     #---------- Ontology Explorer ----------#
+    - name: Ontology Explorer Install dependencies
+      if: steps.changes.outputs.explorer == 'true'
+      run: |
+        python setup.py install
+      working-directory: ./tools/explorer
+    - name: Run pylint on Ontology Explorer
+      if: steps.changes.outputs.explorer == 'true'
+      run: |
+        pylint --rcfile ./styles/pylintrc ./tools/explorer
+    - name: Ontology Explorer Tests
+      if: steps.changes.outputs.explorer == 'true'
+      run: |
+       python -m unittest discover -p '*_test.py' --failfast -v
+      working-directory: ./tools/explorer
+
     #---------- Configuration scoring ----------#
```

Comment:
nit move this last after the rdf generator it is the least that is going to get updated

Reply:
Done. Moved scorer to last.